XamarinCommunityToolkit/Views/GesturePad/Renderer/GesturePadRenderer.tizen.cs

```diff
@@ -0,0 +1,143 @@
+﻿using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using ElmSharp;
+using SkiaSharp;
```

Comment:
@VladislavAntonyuk do you know if we can do it in a native way?
Do we want to add this reference? cc: @jfversluis , @AndreiMisiukevich

Reply:
it should be possible, because SkiaSharp doing it. But I am not Tizen API expert, so I chose SkiaSharp

Reply:
We can ask @rookiejava about this. But I think SkiaSharp is the way to go on Tizen.

Reply:
@pictos I noticed, but the SkiaSharp is only for Tizen, right?

Reply:
@jfversluis, yes

Reply:
Thanks @VladislavAntonyuk for implementing tizen. 👍  
Yes, native also supports simple primitive drawing, but using skiaSharp looks good.